amoro-core/src/main/java/org/apache/amoro/utils/MixedCatalogUtil.java

```diff
@@ -21,28 +21,12 @@
 import org.apache.amoro.TableFormat;
 import org.apache.amoro.api.CatalogMeta;
 import org.apache.amoro.api.TableMeta;
-import org.apache.amoro.io.AuthenticatedFileIO;
```

Comment:
It seems we should rename this class to `CatalogUtil` now.

Reply:
fixed